implementation/src/main/java/io/smallrye/jwt/auth/AbstractBearerTokenExtractor.java

```diff
@@ -60,39 +59,52 @@ public String getBearerToken() {
         final String bearerValue;
 
```

Comment:
Extract token retrieval to separate methods to reduce method complexity (Sonar complexity rating)